op-acceptor/reporting/tree_formatters.go

```diff
@@ -0,0 +1,607 @@
+package reporting
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"strings"
+	"time"
+
+	"github.com/ethereum-optimism/infra/op-acceptor/types"
+	"github.com/ethereum-optimism/infra/op-acceptor/ui"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+// formatDuration formats a duration for display
+func formatDuration(d time.Duration) string {
+	if d < time.Second {
+		return fmt.Sprintf("%dms", d.Milliseconds())
+	}
+	return d.Truncate(time.Millisecond).String()
+}
+
+// TreeHTMLFormatter formats test trees as HTML using the tree structure
+type TreeHTMLFormatter struct {
+	template *template.Template
+}
+
+// NewTreeHTMLFormatter creates a new tree-based HTML formatter
+func NewTreeHTMLFormatter(templateContent string) (*TreeHTMLFormatter, error) {
+	tmpl, err := template.New("tree-report").Funcs(template.FuncMap{
+		"formatDuration": func(d time.Duration) string {
+			if d < time.Second {
+				return fmt.Sprintf("%dms", d.Milliseconds())
+			}
+			return d.Truncate(time.Millisecond).String()
+		},
+		"getStatusClass": func(status types.TestStatus) string {
+			switch status {
+			case types.TestStatusPass:
+				return "pass"
+			case types.TestStatusFail:
+				return "fail"
+			case types.TestStatusSkip:
+				return "skip"
+			case types.TestStatusError:
+				return "error"
+			default:
+				return "unknown"
+			}
+		},
+		"getStatusText": func(status types.TestStatus) string {
+			switch status {
+			case types.TestStatusPass:
+				return "PASS"
+			case types.TestStatusFail:
+				return "FAIL"
+			case types.TestStatusSkip:
+				return "SKIP"
+			case types.TestStatusError:
+				return "ERROR"
+			default:
+				return "UNKNOWN"
+			}
+		},
+		"isTestNode": func(nodeType types.TestTreeNodeType) bool {
+			return nodeType == types.NodeTypeTest || nodeType == types.NodeTypeSubtest
+		},
+		"getIndentClass": func(depth int) string {
+			switch depth {
+			case 0:
+				return "indent-0"
+			case 1:
+				return "indent-1"
+			case 2:
+				return "indent-2"
+			case 3:
+				return "indent-3"
+			default:
+				return fmt.Sprintf("indent-%d", depth)
+			}
+		},
+		"multiply": func(a, b int) int {
```

Comment:
what's this for?

Reply:
No longer used. Removed this and one other. Thanks!